src/atoms/Color/Colors.md

```diff
@@ -1,42 +1,56 @@
 Our brand colors:
 
+    // This is just example code to generate the above palette
```

Comment:
None of these changes actually matter, it's just to have a prettier way of displaying the new palette in the Colors page.
![image](https://user-images.githubusercontent.com/6582067/106980845-edc1f500-672e-11eb-9423-a897e917efe0.png)

Reply:
nah I love this, it _MATTERS_ Ken